src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String fruit;
+    private Operation operation;
+    private int quantity;
+
+    public FruitTransaction(String fruit, Operation operation, int quantity) {
+        this.fruit = fruit;
+        this.operation = operation;
+        this.quantity = quantity;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String code;
+
+        Operation(String code) {
```

Comment:
```suggestion
        private Operation(String code) {
```

Reply:
IDEA tells me that private is reduntant for constructors of enum